draft-ietf-regext-rdap-geofeed.md

```diff
@@ -198,7 +245,7 @@ https://www.iana.org/assignments/media-types/:
 * Interoperability considerations: There are no known interoperability problems regarding this media format.
 * Published specification: This document.
 * Applications that use this media type: Implementations of the Registration Data Access Protocol (RDAP) Extension for
-  Geofeed Data. Furthermore, any application that processes the CSV geofeed data.
+  geofeed Data. Furthermore, any application that processes the CSV geofeed data.
```

Comment:
For consistency, should "Extension for geofeed Data" read "extension for geofeed data"?

Reply:
Woops, thanks, fixed now to match the document title.